index.html

```diff
@@ -286,8 +399,8 @@ <h1 class = "about-heading">About<br>Myself
         <img src="./Images/4downRectangle.png" alt="" class="down-rectangle">
         <img src="./Images/5whiteEllipse.png" alt="" class="white-circle">
         <img src="./Images/end line.png" alt="" class="end-line-image">
-       <h2>Contact me</h2>
-       <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
 
         <form action="https://formspree.io/f/mjvdkngy" method="post">
```

Comment:
- [ ] While you did a good job styling the Form section :1st_place_medal:, I think it would be good to check the background image of it to match as much as possible withe the Figma design

**Your design**

![Screenshot from 2023-01-26 06-52-57](https://user-images.githubusercontent.com/105117832/214829522-41ba660a-9bc0-48cf-90d8-01ecf07be9ef.png)

**Figma design**

![Screenshot from 2023-01-26 06-53-13](https://user-images.githubusercontent.com/105117832/214829582-5f8895d6-e455-44f9-b7c1-babc544fbb20.png)